spec/requests/balance_pages_spec.rb

```diff
@@ -234,6 +234,16 @@
         expect(page).not_to have_text("Expected deposit on")
       end
 
+      it "shows failed payments for creator-owned Stripe Connect payouts" do
```

Comment:
Can we add some more specs e.g. for `displayable_failure_reason` method and PayPal payout failures?